src/Members/PlatformInterface.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the fusonic/webapp package.
+ *
+ * (c) Fusonic GmbH <office@fusonic.net>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Fusonic\WebApp\Members;
+
+/**
+ * List of possible values for the `platform` member.
+ *
+ * <p>
+ * There is no official list of possible values. However, the Web Platform Working Group maintains a list of known
+ * platform values {@link https://github.com/w3c/manifest/wiki/Platforms in their wiki}.
+ *
+ * @package Fusonic\WebApp
+ *
+ * @see PlatformTrait
+ * @see https://github.com/w3c/manifest/wiki/Platforms
+ */
+interface PlatformInterface
+{
+    const PLATFORM_ANDROID = "android";
+    const PLATFORM_IOS = "ios";
+    const PLATFORM_WEB = "web";
+}
```

Comment:
Why have you moved these constants to a separate interface? ATM there is no use-case for such an interface. I'd move it to the `ApplicationConfiguration` class and extract it if needed (maybe in a 2.0).